conftest.py

```diff
@@ -3,6 +3,7 @@
 pytest_plugins = [
     # Plugins
     'pytest_plugins.rerun_rp.rerun_rp',
+    "pytest_plugins.marker_deselection",
```

Comment:
infra_dependent_markers needs to be included here, the markers aren't getting registered with pytest.

Reply:
Also, please keep things consistent and use single quotes.